OpenDreamRuntime/EntryPoint.cs

```diff
@@ -40,7 +40,7 @@ public class EntryPoint : GameServer {
 
         public override void PostInit() {
             _commandSystem = EntitySystem.Get<DreamCommandSystem>();
-            _dreamManager.Initialize();
+            _dreamManager.Initialize(null);
```

Comment:
Have this call pass the cvar instead of having multiple locations default to `testingJson ?? _configManager.GetCVar<string>(OpenDreamCVars.JsonPath)`